doc/src/EntitySpecMeta.dox

```diff
@@ -60,8 +60,8 @@
  * mapped to a manager's internal classification mechanisms. Additional
  * data from the specification's key/value pairs is not persisted beyond
  * the lifetime of the related API calls in any literal form. If a host
- * wishes to persist any other information about an entity, use the @ref
- * attributes mechanism instead.
+ * wishes to persist any other information about an entity, use the
+ * @needsref attributes mechanism instead.
```

Comment:
[question] Should this be
```suggestion
 * @needsref traits mechanism instead.
```
(similarly below)

Reply:
Indeed, there is a whole `@section` for Attributes below.

Ah, I should read the description
> The only documentation left un-altered is the "Entities, Specifications and Attributes" page, which is tackled independently under https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/249.

So I take it the `@needsref`s in this file are just so we can't forget to update later?

Reply:
Yeah - and to stop the docs build failing